src/sqlancer/postgres/PostgresProvider.java

```diff
@@ -325,17 +325,24 @@ protected void prepareTables(PostgresGlobalState globalState) throws Exception {
     private String getCreateDatabaseCommand(PostgresGlobalState state) {
         StringBuilder sb = new StringBuilder();
         sb.append("CREATE DATABASE " + databaseName + " ");
-        if (Randomly.getBoolean() && ((PostgresOptions) state.getDbmsSpecificOptions()).testCollations) {
+        if (((PostgresOptions) state.getDbmsSpecificOptions()).testCollations) {
             if (Randomly.getBoolean()) {
-                sb.append("WITH ENCODING '");
-                sb.append(Randomly.fromOptions("utf8"));
-                sb.append("' ");
-            }
-            for (String lc : Arrays.asList("LC_COLLATE", "LC_CTYPE")) {
-                if (!state.getCollates().isEmpty() && Randomly.getBoolean()) {
-                    sb.append(String.format(" %s = '%s'", lc, Randomly.fromList(state.getCollates())));
+                if (Randomly.getBoolean()) {
+                    sb.append("WITH ENCODING '");
+                    sb.append(Randomly.fromOptions("utf8"));
+                    sb.append("' ");
+                }
+                for (String lc : Arrays.asList("LC_COLLATE", "LC_CTYPE")) {
+                    if (!state.getCollates().isEmpty() && Randomly.getBoolean()) {
+                        sb.append(String.format(" %s = '%s'", lc, Randomly.fromList(state.getCollates())));
+                    }
                 }
+                sb.append(" TEMPLATE template0");
             }
+        } else {
+            sb.append("WITH ENCODING '");
+            sb.append(Randomly.fromOptions("utf8"));
```

Comment:
Here, you can directly use "utf8" rather than `Randomly.fromOptions("utf8")`.

Reply:
Done. I was initially tempted to conflate the options, but saw another place use `fromOptions("utf8")` and erred to did that instead.

Anyway, have pushed another commit with conflated options.